modules/ting_reference/ting_reference.install

```diff
@@ -128,59 +128,94 @@ function ting_reference_update_7001() {
 
   db_create_table('ting_reference', $schema['ting_reference']);
```

Comment:
And next time instead duplicating schema definition use something like this
https://github.com/easyddb/thisted_event_import/blob/develop/thisted_event_import.install#L63